eng/Update-PkgVersion.ps1

```diff
@@ -0,0 +1,145 @@
+<#
```

Comment:
Remember this filename change will need a matching change in the release pipeline, or if you want to do safer you can make the Version-Increment script a simple wrapper around this one.

Reply:
I have a PR out for the name change.